dataRetriever/dataPool/headersCache/headersPool_test.go

```diff
@@ -0,0 +1,342 @@
+package headersCache_test
+
+import (
+	"fmt"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever/dataPool/headersCache"
+	"github.com/stretchr/testify/assert"
+)
+
+func createASliceOfHeaders(numHeaders int, shardId uint32) ([]block.Header, [][]byte) {
```

Comment:
Move helper functions at the end of the test.

Reply:
done.